README.md

```diff
@@ -24,6 +69,11 @@ All private keys used in the library are for testing only and not used in any pr
   - Optionally specify `federate.hsmConfigs.keyPath` to your local keys, if you happen to be testing different keys or your keys are located in a different path.
 4. 
 	-  Run `npm test` to run testcases without interruption or `npm run test-fail-fast` to stop execution at first testcase failure.
+  - `regtest.js` has by default not using the following mapping:
+    - federate1: no hsm signer
+    - federate2 and federate3: hsm1
+    - federate4 and federate5: hsm2 (modify using the same logic that in federate2 and 3 for changing to hsm1 in case of hsm2 not working).
+  
```

Comment:
We Should Update regtest.js To Use fed1 - keyfile, fed2 & fed3 - hsm1, fed4 & fed5 - hsm2

Reply:
The pipeline.conf Is Configured As I Described Above, Hence Line 72 To 75 Can Be Removed